rust/crd/src/affinity.rs

```diff
@@ -93,6 +93,9 @@ mod test {
                             namespace_selector: None,
                             namespaces: None,
                             topology_key: "kubernetes.io/hostname".to_string(),
+                            // These fields are only available starting with kubernetes 1.30
+                            mismatch_label_keys: None,
+                            match_label_keys: None,
```

Comment:
We could default here for all "None" options (thats what we did in most operators).

Reply:
done https://github.com/stackabletech/spark-k8s-operator/pull/401/commits/9157ebe34443c4278849ff5415cabb28077be378